src/pseudopeople/configuration/interface.py

```diff
@@ -6,29 +6,45 @@
 
 def get_config(overrides: Union[Path, str, Dict] = None) -> Dict:
     """
-    Function that returns the pseudopeople configuration including all default values.
-    To get the default probability of nonresponse in the Decennial Census dataset:
+    Function that returns the pseudopeople configuration containing all
+    default values. To get the default probability of nonresponse in the
+    Decennial Census dataset:
 
     .. code-block:: pycon
 
         >>> import pseudopeople as psp
         >>> psp.get_config()['decennial_census']['row_noise']['do_not_respond']
         {'row_probability': 0.0145}
 
-    To view that same part of the configuration after applying a user override:
+    To view that same part of the configuration after applying a user
+    override:
 
     .. code-block:: pycon
 
         >>> overrides = {'decennial_census': {'row_noise': {'do_not_respond': {'row_probability': 0.1}}}}
         >>> psp.get_config(overrides)['decenial_census']['row_noise']['do_not_respond']
         {'row_probability': 0.1}
 
-    :param overrides: An optional override to the default configuration. Can be
-        a path to a configuration YAML file or a dictionary. Passing a sentinel value
-        of psp.NO_NOISE will override default values and return a configuration
-        where all noise levels are set to 0.
-    :return: Dictionary of the config.
-    :raises ConfigurationError: An invalid configuration is passed with overrides.
+    :param overrides:
+
+        An optional set of overrides to the default configuration. Can
+        be a (nested) Python dictionary mapping noise type parameters to
+        the desired override values, a path to a YAML file with the same
+        :ref:`nested structure <configuration_structure>`, or the
```

Comment:
Ah, good point. No, I don't think any of the other docstrings contain references to other sections of the docs. Maybe I should add another sentence that says something like, "see the :ref:`configuration structure <configuration_structure>` section of the documentation for more details," so that it would be clear that this is referencing the documentation. I could even use an explicit hyperlink reference instead of `:ref:` so that a user reading the docstring directly would have the URL, but then that runs the risk of getting out of date.

Reply:
I added the quoted phrase above in parentheses, leaving the `:ref:` command, not a URL.